docs/zopebook/Sessions.rst

```diff
@@ -1435,3 +1456,120 @@ attempt to provide application-level conflict resolution to reduce the
 limitations imposed by conflict errors NOTE: to take advantage of this feature,
 you must store your transient object container in a storage such as FileStorage
 or TemporaryStorage which supports application-level conflict resolution.
+
+Alternative Server Side Session Backends for Zope 4+
+====================================================
+
+To re-enable server side sessions on Zope 4+, you have two  ways to go about 
+it. You can use a seperate session server, most likely using 
+`Memcached <https://memcached.org>`_, or place the session storage in either a 
+``<filestorage>`` or ``<mappingstorage>`` backed ZODB.
+
+Use of an alternative session server
+++++++++++++++++++++++++++++++++++++
+
+There are two projects that enable you to use 
+`Memcached <https://memcached.org>`_ in Zope projects. This is the recommended 
+way to use server side sessions.
+
+- `Products.mcdutils <https://pypi.org/project/Products.mcdutils/>`_ is a drop 
+  in replacement for `tempstorage <https://github.com/zopefoundation/tempstorage>`_ 
```

Comment:
The sentence is misleading. `Products.mcdutils` is a drop-in replacement for the Zope 2 session implementation. Not for `tempstorage`, which is just one small implementation detail of that session mechanism.

Reply:
[x] done (push to come)